### PR TITLE
jca_issue_1049

### PR DESCRIPTION
**Pull request by jcohenadad**

### CODE REVIEW

File: scripts/sct_register_multimodal.py [hunk 1]
```diff
@@ -428,10 +472,17 @@ def main(args=None):
 
     # Generate output files
     sct.printv('\nGenerate output files...', verbose)
+    # src_reg
```
Comment (peristeri) [inline scripts/sct_register_multimodal.py]: I would either provide a more meaningful message on line 473 or avoid it all together. The one word comments are noisy and provide no value.

File: scripts/sct_register_multimodal.py [hunk 2]
```diff
@@ -428,10 +472,17 @@ def main(args=None):
...
+    # warp
+    if fname_output_warp == '':
+        fname_output_warp = path_out+'warp_'+file_src+'2'+file_dest+'.nii.gz'
+    sct.generate_output_file(path_tmp+'warp_src2dest.nii.gz', fname_output_warp, verbose)
+    if generate_warpinv:
+        # dest_reg
+        fname_dest2src = sct.generate_output_file(path_tmp+'dest_reg.nii', path_out+file_dest+'_reg'+ext_dest, verbose)
+        # warpinv
+        sct.generate_output_file(path_tmp+'warp_dest2src.nii.gz', path_out+'warp_'+file_dest+'2'+file_src+'.nii.gz', verbose)
     # sct.generate_output_file(path_tmp+'/warp_dest2src.nii.gz', path_out+'warp_dest2src.nii.gz')
```
Comment (peristeri) [inline scripts/sct_register_multimodal.py]: You can cleanup all commented code.

File: scripts/sct_register_multimodal.py (hunk 1 above)
Comment (jcohenadad) [inline scripts/sct_register_multimodal.py]: thanks-- i've pushed modifs

File: scripts/sct_register_multimodal.py (hunk 2 above)
Comment (jcohenadad) [inline scripts/sct_register_multimodal.py]: done

**Comment by jcohenadad** [comment]
batch_processing:
~~~
t2/CSA:   76.753884, 2.049822
mt/MTR:   51, white matter, 387.801068, 33.744919, 0.000000
mt/CSA:   20.052757, 1.286250
dmri/FA:  4, WM left lateral corticospinal tract, 22.799808, 0.734177, 0.000000
dmri/FA:  5, WM right lateral corticospinal tract, 20.828494, 0.790546, 0.000000
~~~

**Pull request merged**